src/main/java/io/logz/jmx2graphite/Jmx2GraphiteConfiguration.java

```diff
@@ -8,16 +8,20 @@
 import java.net.UnknownHostException;
 import java.util.concurrent.TimeUnit;
 import java.util.logging.Level;
+import java.util.regex.Pattern;
 
 /**
  * @author amesika
  */
 public class Jmx2GraphiteConfiguration {
-    
+
+    private static final String WHITE_LIST_ACCEPT_ALL_REGEX = ".*";
+    private static final String BLACK_LIST_ACCEPT_ALL_REGEX = "$a";
```

Comment:
Why do you mean by Black List which accepts all , regular expression?

Reply:
Is "BLACK_LIST_FILTER_NOTHING_REGEX" more clear?